### PR TITLE
KevS-SK1293-KFT0010282-Update-Configuring-Perso_Pro_Perf-Webhosting

### PR DESCRIPTION
**Pull request by Kazarus4**
Updates (FRFR only): 

- Date of last update;
- Correcting typos;
- Correcting technical mistakes;
- Gave more infos and explanations to clarify these technical specificities.

Need tech and peer review before FRCA duplication and merge on develop.

### CODE REVIEW

**Comment by Kazarus4** [comment]
need peer rv by @BrenKt , tech review already done.

**Review by BrenKt — APPROVED** [review]
Review done

File: pages/web_cloud/web_hosting/hosting_technical_specificities/guide.fr-fr.md [hunk 1]
```diff
@@ -18,78 +20,96 @@ updated: 2022-11-18
...
-- En cas d'abus ou de risque avéré, votre service sera suspendu et vous serez notifié par e-mail de la suspension de celui-ci. Que faire en cas de compte bloqué pour spam ? Consultez [notre guide](/pages/web_cloud/email_and_collaborative_solutions/troubleshooting/locked_for_spam).
+> [!primary]
+>
+> Ces limitations concernent **uniquement** les e-mails émis à l'aide de la fonction *mail()* de PHP et ne concerne pas les autres offres e-mail (envoi SMTP).
+>
+
+À l'exception des suspicions de spamming ou de phishing, l'expédition de vos e-mails pourra être différée. Vos e-mails seront conservés dans une file d'attente jusqu'à ce que le nombre d'e-mails envoyés au cours de l'heure écoulée soit inférieur au quota.
+
+En cas d'abus ou de piratage, une partie ou l'ensemble de votre service pourra être suspendu (conformément aux CGU/CGV et Conditions particulières de votre offre). Vous serez notifié par e-mail de la suspension de celui-ci. Dans ce cas de fugure, appuyez-vous des guides suivants :
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/hosting_technical_specificities/guide.fr-fr.md]:
```suggestion
En cas d'abus ou de piratage, une partie ou l'ensemble de votre service pourra être suspendu (conformément aux CGU/CGV et Conditions particulières de votre offre). Vous serez notifié par e-mail de la suspension de celui-ci. Dans ce cas de figure, appuyez-vous les guides suivants :
```

File: pages/web_cloud/web_hosting/hosting_technical_specificities/guide.fr-ca.md [hunk 2]
```diff
@@ -1,12 +1,14 @@
 ---
-title: 'Spécificités techniques liées aux hébergements mutualisés'
-excerpt: 'Vous trouverez dans ce guide différentes informations et spécificités techniques liées aux hébergements mutualisés.'
-updated: 2022-11-18
+title: "Spécificités techniques liées aux hébergements mutualisés"
+excerpt: "Découvrez dans ce guide différentes informations et spécificités techniques liées aux hébergements Web"
+updated: 2023-11-16
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/hosting_technical_specificities/guide.fr-ca.md:4]: ### 
```suggestion
updated: 2023-11-16
```

Comment (Kazarus4) [inline pages/web_cloud/web_hosting/hosting_technical_specificities/guide.fr-ca.md]:
```suggestion
updated: 2023-11-24
```

File: pages/web_cloud/web_hosting/hosting_technical_specificities/guide.fr-fr.md [hunk 3]
```diff
@@ -1,12 +1,14 @@
 ---
-title: 'Spécificités techniques liées aux hébergements mutualisés'
-excerpt: 'Vous trouverez dans ce guide différentes informations et spécificités techniques liées aux hébergements mutualisés.'
-updated: 2022-11-18
+title: "Spécificités techniques liées aux hébergements mutualisés"
+excerpt: "Découvrez dans ce guide différentes informations et spécificités techniques liées aux hébergements Web"
+updated: 2023-11-16
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/hosting_technical_specificities/guide.fr-fr.md]:
```suggestion
updated: 2023-11-24
```

**Pull request merged**